constants.gradle

```diff
@@ -13,7 +13,7 @@
 // limitations under the License.
 project.ext {
     // ExoPlayer version and version code.
-    releaseVersion = '2.16.1'
+    releaseVersion = '2.16.1.6'
```

Comment:
ℹ️ We cannot trick the version the same as what we did for Jitpack and we need to add our `.x` number in the end.